vm/src/pyobject.rs

```diff
@@ -262,6 +264,7 @@ impl AttributeProtocol for PyObjectRef {
             PyObjectKind::Module { name: _, ref dict } => dict.contains_key(attr_name),
             PyObjectKind::Class { name: _, ref dict } => dict.contains_key(attr_name),
             PyObjectKind::Instance { ref dict } => dict.contains_key(attr_name),
+            PyObjectKind::List { elements: _ } => self.typ().has_attr(attr_name),
```

Comment:
```has_attr``` should return false. The list itself doesn't have attributes (only its class does).

Reply:
Agreed!